cmd/kots/cli/install.go

```diff
@@ -399,6 +400,9 @@ func InstallCmd() *cobra.Command {
 	cmd.Flags().MarkHidden("with-dockerdistribution")
 	cmd.Flags().MarkHidden("storage-base-uri-plainhttp")
 
+	cmd.Flags().Bool("ensure-rbac", true, "when set, kots will create the roles and rolebindings necessary to manage applications")
+	cmd.Flags().MarkHidden("ensure-rbac")
```

Comment:
does this need to be hidden?

Reply:
I think we should keep this hidden and possibly remove it once we have a proper fix

Reply:
tbh i think this is a pretty good solution. we can do a better job with minimal rbac support but this seems different.